crates/stages/stages/src/stages/utils.rs

```diff
@@ -124,14 +124,14 @@ where
 
     // observability
     let total_entries = collector.len();
-    let interval = (total_entries / 100).max(1);
+    let interval = (total_entries / 1000).max(1);
```

Comment:
this now logs more often, every 0.1% blocks. We need the opposite result.
```suggestion
    let interval = (total_entries / 10).max(1);
```

Reply:
Aha yes, nice catch !